BitwardenShared/UI/Platform/Settings/Settings/About/AboutProcessorTests.swift

```diff
@@ -133,11 +141,53 @@ class AboutProcessorTests: BitwardenTestCase {
         XCTAssertTrue(errorReporter.isEnabled)
     }
 
-    /// `receive(_:)` with action `.versionTapped` copies the version string to the pasteboard.
-    func test_receive_versionTapped() {
+    /// `receive(_:)` with action `.versionTapped` copies the copyright, the version string
+    /// and device info to the pasteboard when no additional info is provided.
+    func test_receive_versionTapped_noAdditionalInfo() {
         subject.receive(.versionTapped)
-        let text = subject.state.copyrightText + "\n\n" + subject.state.version
-        XCTAssertEqual(pasteboardService.copiedString, text)
+        XCTAssertEqual(
+            pasteboardService.copiedString,
+            """
+            © Bitwarden Inc. 2015-2024
+
+            Version: 2024.6.0 (1)
+
+            -------- Device --------
+
+            Model: iPhone14,2
+            OS: iOS 16.4
+            """
+        )
+        XCTAssertEqual(subject.state.toast?.text, Localizations.valueHasBeenCopied(Localizations.appInfo))
+    }
+
+    /// `receive(_:)` with action `.versionTapped` copies the copyright, the version string,
+    /// device info and the additional info to the pasteboard when it's provided.
+    func test_receive_versionTapped_withAdditionalInfo() {
+        aboutAdditionalInfo.ciBuildInfo = [
+            "Repository": "www.github.com/bitwarden/ios",
+            "Branch": "tesBranch",
+        ]
+
+        subject.receive(.versionTapped)
+        XCTAssertEqual(
+            pasteboardService.copiedString,
+            """
+            © Bitwarden Inc. 2015-2024
+
+            Version: 2024.6.0 (1)
+
+            -------- Device --------
+
+            Model: iPhone14,2
+            OS: iOS 16.4
+
+            ------- CI Info --------
+
+            Branch: tesBranch
```

Comment:
🎨 Typo on the branch name, and our normal convention is hyphenated words, I think, so it might be nice to match that convention?

```suggestion
            Branch: test-branch
```